addons/sourcemod/scripting/vsh/bosses/boss_horsemann.sp

```diff
@@ -120,6 +120,8 @@ public void Horsemann_OnSpawn(SaxtonHaleBase boss)
 	264: melee range multiplier (tf_weapon_sword have 37% extra range)
 	551: special taunt
 	*/
+
+	Horsemann_CreateEyeGlow(boss);
```

Comment:
Can the eye glow entity stack if boss were to keep suiciding during setup time? Don't know if `Horsemann_Create` would work there, or would have to kill existing glow ents.